packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -81,10 +83,16 @@ function App() {
 
         updateProjectSettings({ menuLogo: prefixFileUrlWithBackendUrl(menuLogo) });
 
-        if (uuid) {
+        if (uuid && currentToken) {
           const {
             data: { data: properties },
-          } = await axios.get(`${strapi.backendURL}/admin/telemetry-properties`);
+          } = await axios.get(`${strapi.backendURL}/admin/telemetry-properties`, {
+            headers: {
+              Authorization: `Bearer ${currentToken}`,
+              Accept: 'application/json',
+              'Content-Type': 'application/json',
+            },
+          });
```

Comment:
@simotae14 Where are we with the new fetch hook? This would be a prime example to use it :)

Reply:
He's on holiday but it's ready to merge, so maybe we should just merge it?

Reply:
That would be great - I'll do it. @fdel-car Would be up to refactoring this part a bit using https://github.com/strapi/strapi/pull/14759?

Reply:
Yes, actually I did it with the useFetchClient hook in the first place a couple of weeks ago, but as it wasn't getting merged I re-implemented it without it.